src/Compilers/CSharp/Portable/Parser/LanguageParser_Patterns.cs

```diff
@@ -581,15 +586,26 @@ private PropertyPatternClauseSyntax ParsePropertyPatternClause()
         private SubpatternSyntax ParseSubpatternElement()
```

Comment:
It looks like this also affects the parsing of positional patterns. Can we check that with member access syntax and verify it's still an error, if not covered already?

Reply:
I was going to leave this for another PR, if it's ok. (I added prototype comments wherever that need to be checked)

Reply:
PROTOTYPE marker is fine.

Reply:
I couldn't find if/where a PROTOTYPE marker was added to track this question. Was this addressed?

Reply:
Most of prototype comments on NameColon usages are meant to refer to this test gap (should check ExpressionColon instead and adjust the code accordingly)